lib/OpenQA/WebAPI/Plugin/AuditLog.pm

```diff
@@ -39,7 +39,14 @@ sub register {
     my ($self, $app, $reactor) = @_;
 
     # register for events
-    for my $event (@table_events, @job_events, @jobgroup_events, @jobtemplate_events, @user_events, @asset_events, @iso_events, @worker_events, @needle_events) {
+    my @events = (@table_events, @job_events, @jobgroup_events, @jobtemplate_events, @user_events, @asset_events, @iso_events, @worker_events, @needle_events);
+    # filter out blacklisted events
+    my @blacklist = split / /, $app->config->{audit}{blacklist};
+    while (@blacklist) {
```

Comment:
that looks to me like a very complicated way to write for my $e (@blacklist) - or am I missing something?